docs/_partners/integration-quality.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Assessing and Improving your Integration's Quality
+order: 4
+layout: post-toc
+redirect_from: /partners/
+---
+
+# Assessing and Improving your Integration's Quality
+
+Integration quality on Zapier boils down to two main pillars: **Health** and **Depth**.
+
+1. **Health** - does the integration allow users to set up Zaps and keep them running reliably?
+2. **Depth** - does the integration have all the functionality to allow users to automate what they want through their Zaps?
+
+We’ll examine the main metrics around integration health and explore what they could be telling us about an integration’s quality level:
+
+* **Errors** 
+* **Zap Activation Rate**
+* **Bugs**
+* **Active User Retention**
+
+Utilize the various tools and resources available in the developer platform, such as [Bugs and Feature Requests](https://platform.zapier.com/partners/feature-requests-bugs#how-to-monitor-feature-requests-and-bugs), [Monitoring](https://platform.zapier.com/docs/testing#monitoring), and your integration’s Dashboard to monitor and analyze these metrics and your integration’s health. Let’s dig in!
+
+
+## Errors
+
+Errors can cause pain for users at two vital points:
+
+1. During the Zap setup process, stopping them from enabling and activating their Zaps
+2. During the Zap’s runtime, once it has been turned on, preventing the Zap from completing all actions successfully
+
+### Are all errors “bad”?
+
+We don’t want integrations throwing errors unnecessarily, but not all errors are necessarily bad. It’s perfectly acceptable for your integration to throw errors, as long as [they are meaningful](https://platform.zapier.com/partners/integration-review-guidelines#7-error-handling) in stating the problem and ideally the resolution, and [handled correctly](https://platform.zapier.com/cli_docs/docs#error-handling). For example, warnings due to user error or duplications are appropriate to throw and not an issue of the integration itself. So, it’s helpful to identify which errors are actual errors of the integration’s functionality.
+
+### What we have learned
+
+Spikes in errors should be monitored as leading indicators of problems users are facing within your integration. The more descript and thorough [error handling](https://platform.zapier.com/cli_docs/docs#error-handling) your API and integration have in place, the easier it will be for users to resolve their own issues and for Zapier’s support team to assist with debugging.
```

Comment:
Possible typo: `descript` ➡️  `descriptive`

Reply:
Actually maybe that's intentional.. But when I read the whole sentence it sounds weird to me 🙈

Reply:
Regardless of intention, if it reads unclear, that's great feedback! I'll update it to `descriptive`.